src/server.h

```diff
@@ -1031,6 +1032,9 @@ typedef struct rdbLoadingCtx {
     functionsLibCtx *functions_lib_ctx;
 } rdbLoadingCtx;
 
+typedef sds (*rdbAuxFieldEncoder)(int flags);
+typedef int (*rdbAuxFieldDecoder)(int flags, void *p);
```

Comment:
```suggestion
typedef int (*rdbAuxFieldDecoder)(int flags, sds *p);
```
By the implementation this looks like an sds.

Reply:
My reading is that it is guaranteed to be a null-terminated C string but not always an sds string? And I only need a c string. Will change it to `char *`